cloud-agnostic/core/src/NamedDependency.ts

```diff
@@ -30,6 +37,15 @@ export abstract class NamedDependency extends Dependency {
     serviceIdentifier: interfaces.ServiceIdentifier<T>,
     instanceName: string
   ): void {
+    container
```

Comment:
we bind the same instance twice. why exactly is this needed?

Reply:
The previous binding works only with `@named` or `getNamed()`. This new binding works with `@multiInject` and `getAll()`.